src/api/helpers/date/index.ts

```diff
@@ -55,3 +57,19 @@ export const getThirtyDaysBeforeNow = (): Date => {
 
   return new Date(result);
 };
+
+/**
+ * dateIsInThePast
+ * Is the time now after a specified date
+ * @param {Date} specifiedDate
+ * @returns {Boolean}
+ */
+export const dateIsInThePast = (specifiedDate?: Date) => {
+  if (specifiedDate) {
+    const now = new Date();
+
+    return isAfter(now, specifiedDate);
```

Comment:
`targetDate` instead of `specifiedDate`?

Reply:
Also should this param not always be required?

Feels misleading if it returns `false` ("not in the past") if no date is provided